src/jit/gentree.h

```diff
@@ -1459,9 +1454,10 @@ struct GenTree
             case GT_INTRINSIC:
             case GT_LEA:
             case GT_STOREIND:
-            case GT_INITBLK:
-            case GT_COPYBLK:
-            case GT_COPYOBJ:
+            case GT_BLK:
+            case GT_OBJ:
```

Comment:
Yes, but they are not GTK_BINOP, so they won't reach here. However, GT_STOREIND shouldn't be here because it *must* have an op2, which is the data to be stored. So I have deleted it as well as the others that I added.

---
In reply to: [77283459](https://github.com/dotnet/coreclr/pull/7027#discussion_r77283459) [](ancestors = 77283459,77283180,77282926)